WebDriverAgentLib/Utilities/FBImageIOScaler.h

```diff
@@ -42,7 +42,8 @@ extern const CGFloat FBMaxCompressionQuality;
 
  @param image The source image data
  @param uti Either kUTTypePNG or kUTTypeJPEG
- @param rect The cropping rectangle. Could be CGRectNull to avoid cropping
+ @param rect The cropping rectange for the screenshot. The value is expected be non-scaled one.
```

Comment:
actually here the value of rect is expected to be already scaled since cropping is happening `after` scaling/orientation change